app/lib/diograph-api.ts

```diff
@@ -14,7 +14,9 @@ export class DiographApi {
     }
   }
 
-  private static baseUrl = localStorage.get("endpoint")
+  // private static baseUrl = "http://diory-server.herokuapp.com/v1/";
+  private static baseUrl = "http://localhost:3000/v1/";
+  // private static baseUrl = localStorage.getItem("endpoint")
```

Comment:
Remember to revert this ` private static baseUrl = localStorage.getItem("endpoint")` back before publishing...